src/migrations/index.js

```diff
@@ -46,6 +46,28 @@ const migrations = [
        table.string('reason_code').nullable().default('')
      })
    }
+  },
+  { auto: true, //4
+    date: '2017-10-23',
+    migrate: async function migrate() {
+      await r.knex.schema.alterTable('message', (table) => {
+        table.timestamp('queued_at');
+        table.timestamp('sent_at');
+        table.timestamp('service_response_at');
+      })
+      console.log('added action timestamp columns to message')
+    }
+  },
+  { auto: true, //5
+    date: '2017-10-23',
+    migrate: async function migrate() {
+      await r.knex.schema.createTable('log', (table) => {
```

Comment:
I'm a little nervous about a race condition here -- `createTable` will also end up being run in your line if server/models/log.js:
`const Log = thinky.createModel('log', type.object().schema({`
Maybe just do `r.knex.schema.createTableIfNotExists` instead of `createTable` and we should be fine.